src/components/timecard/start-finish-time-input/StartFinishTimeInput.jsx

```diff
@@ -29,22 +30,34 @@ const StartFinishTimeInput = ({
     setErrorMessages(relevantErrorMessages);
   };
 
+  const sortErrors = () => {
+    const errorKeys = [];
+    const errorMessageKeys = Object.keys(errors);
+    if (errorMessageKeys.includes('shift-start-time')) {
+      errorKeys.push('shift-start-time');
+    }
+    if (errorMessageKeys.includes('shift-finish-time')) {
+      errorKeys.push('shift-finish-time');
+    }
+    return errorKeys;
+  };
+
   return (
     <div
       className={`govuk-form-group ${
         errorMessages.length > 0 && 'govuk-form-group--error'
       }`}
     >
       <div className="govuk-grid-row">
-        {errorMessages &&
-          errorMessages.map((error, i) => (
+        {sortErrors() &&
```

Comment:
I think you can get away with just  `sortErrors().map` and remove the first `sortErrors()` check as  `sortErrors().map` should just not render anything if the array is empty.

Reply:
I've removed the extra check for sortErrors